### PR TITLE
moved non stomp critical resource files

### PR DESCRIPTION
**Pull request by jrgnicho**
This PR is a followup to #86 .  It mainly copies or moves files needed by the `industrial_moveit_benchmarking` package from their location into this package. 
Only the cartesian and stomp valgrind nodes seem to work, every test in the joint_valgrind node fails.  However CI should not be affected by this change as these executables aren't unit test.

### CODE REVIEW

File: constrained_ik/src/constrained_ik.cpp
Comment (gavanderhoorn) [inline constrained_ik/src/constrained_ik.cpp:32]: This change (and other changes to this file) don't seem to belong in this PR?

File: industrial_moveit_benchmarking/launch/cartesian_valgrind.launch
Comment (gavanderhoorn) [inline industrial_moveit_benchmarking/launch/cartesian_valgrind.launch:7]: These are no longer private params?

File: industrial_moveit_benchmarking/launch/joint_interpolated_valgrind.launch
Comment (gavanderhoorn) [inline industrial_moveit_benchmarking/launch/joint_interpolated_valgrind.launch]: Seems like a layout change only?

File: industrial_moveit_benchmarking/launch/stomp_valgrind.launch
Comment (gavanderhoorn) [inline industrial_moveit_benchmarking/launch/stomp_valgrind.launch:6]: Same question as for `cartesian_valgrind.launch`.

File: industrial_moveit_benchmarking/resources/config/test_kr210.srdf
Comment (gavanderhoorn) [inline industrial_moveit_benchmarking/resources/config/test_kr210.srdf:1]: Is this a new file, or was this moved from somewhere? If moved: did you use `git mv`? Git doesn't seem to understand this was moved from somewhere.

File: industrial_moveit_benchmarking/src/stomp_valgrind.cpp
Comment (gavanderhoorn) [inline industrial_moveit_benchmarking/src/stomp_valgrind.cpp:156]: This seems like an unrelated change?

**Review by gavanderhoorn — APPROVED** [review]
Apart from some comments this looks ok.

Question: your intent seems to be to have all `stomp_*` packages moved to `ros-industrial/stomp_ros` (including the testing packages), is that correct?

**Comment by gavanderhoorn** [comment]
> This PR is a followup to #86.

I think we need to merge this PR before we merge/continue with #86, no?

File: industrial_moveit_benchmarking/resources/config/test_kr210.srdf
Comment (jrgnicho) [reply industrial_moveit_benchmarking/resources/config/test_kr210.srdf:1]: It was copied from stomp_test_kr210_moveit_config

File: constrained_ik/src/constrained_ik.cpp
Comment (jrgnicho) [reply constrained_ik/src/constrained_ik.cpp:32]: The cartesian_valgrind node failed prior to this change.   It is unrelated to the intent of this PR but since it was a small change I thought it wouldn't hurt to make it.

File: industrial_moveit_benchmarking/launch/cartesian_valgrind.launch
Comment (jrgnicho) [reply industrial_moveit_benchmarking/launch/cartesian_valgrind.launch:7]: Right, it seems that at some point a change was made were it no longer looked for the parameters under the node's private namespace.

File: industrial_moveit_benchmarking/launch/joint_interpolated_valgrind.launch
Comment (jrgnicho) [reply industrial_moveit_benchmarking/launch/joint_interpolated_valgrind.launch]: Yes, I momentarily added `output="screen"` and change the layout for better readability but didn't restore it to its original form when I took it out.

File: industrial_moveit_benchmarking/launch/stomp_valgrind.launch
Comment (jrgnicho) [reply industrial_moveit_benchmarking/launch/stomp_valgrind.launch:6]: I ran into the same issue where the node wasn't loading the parameters from the private namespace.

File: industrial_moveit_benchmarking/src/stomp_valgrind.cpp
Comment (jrgnicho) [reply industrial_moveit_benchmarking/src/stomp_valgrind.cpp:156]: The fact that it printed an error seemed confusing to me as it wasn't really an error, just a workaround to force the message to print on the console and so I changed it to print the result as a regular message.

File: constrained_ik/src/constrained_ik.cpp
Comment (gavanderhoorn) [reply constrained_ik/src/constrained_ik.cpp:32]: Ok.

It might be good to split these kind of changes into separate commits, so that file & directory layout changes are not mixed with content changes.

I'll leave it to you whether you'd want to change this PR or not.

File: industrial_moveit_benchmarking/launch/joint_interpolated_valgrind.launch
Comment (gavanderhoorn) [reply industrial_moveit_benchmarking/launch/joint_interpolated_valgrind.launch]: Could you please revert this change? Let's not introduce non-functional changes in files when we're moving things around (keeps the PR simpler and more on-topic).

**Comment by jrgnicho** [comment]
> I think we need to merge this PR before we merge/continue with #86, no?

Yes

**Comment by jrgnicho** [comment]
> Question: your intent seems to be to have all stomp_* packages moved to ros-industrial/stomp_ros (including the testing packages), is that correct?

Well yes, since I saw that all the `stomp_ ...` packages are in https://github.com/ros-industrial/stomp_ros.  I'm not set on that however, it may make sense to keep the `stomp_test `packages in this repo and rename them to something more generic like `industrial_moveit_test...`

**Comment by gavanderhoorn** [comment]
> > Question: your intent seems to be to have all stomp_* packages moved to ros-industrial/stomp_ros (including the testing packages), is that correct?
> 
> Well yes, since I saw that all the `stomp_ ...` packages are in https://github.com/ros-industrial/stomp_ros.

The current contents and layout of `ros-industrial/stomp_ros` was just an initial prototype. We can change it to be / host whatever.

> I'm not set on that however, it may make sense to keep the `stomp_test `packages in this repo and rename them to something more generic like `industrial_moveit_test...`

Ok, we could do that.

Are those packages specifically for testing STOMP? Or could they also be used to test other things?

**Comment by jrgnicho** [comment]
> Ok, we could do that.
Are those packages specifically for testing STOMP? Or could they also be used to test other things?

They can also be used for the other planners, in fact that's what is done in `industrial_moveit_benchmarking`

File: industrial_moveit_benchmarking/launch/joint_interpolated_valgrind.launch
Comment (jrgnicho) [reply industrial_moveit_benchmarking/launch/joint_interpolated_valgrind.launch]: I reverted this

**Comment by gavanderhoorn** [comment]
> They can also be used for the other planners, in fact that's what is done in `industrial_moveit_benchmarking`

Ah, well then your suggestion makes a lot of sense, and hosting things not specifically tied to STOMP in the `stomp_ros` repository would seem to be strange.

My suggestion would then be to move only the non-test `stomp_*` packages to `ros-industrial/stomp_ros` and keep the rest here.

Would you want to change this PR to implement what you suggested? Or do that in another PR?

**Comment by jrgnicho** [comment]
> Would you want to change this PR to implement what you suggested? Or do that in another PR?

Great, I'll start another PR for this then

**Comment by jrgnicho** [comment]
@gavanderhoorn the requested changes have been made.

**Comment by gavanderhoorn** [comment]
> @gavanderhoorn the requested changes have been made.

So I guess you'll submit a follow-up PR to move more things around / rename them ;)

**Comment by jrgnicho** [comment]
> So I guess you'll submit a follow-up PR to move more things around / rename them ;)

Yes

**Comment by jrgnicho** [comment]
@gavanderhoorn are you OK with merging this?

**Comment by gavanderhoorn** [comment]
Yep. Just did.

If you can submit the PR where we move thing around / rename them that would be great.

**Pull request merged**
